### ISSUE TITLE
Evaluate key command change

### ISSUE THREAD

**Issue by algomusic** [noahingham/extempore-atom#1]
I suggest a change to the Evaluate key command to become Alt-E. This is for several reasons. 1) The current Alt-W is very similar (on a Mac) to Cmd-W which closes the current Atom buffer - making this mistake in a live coding concert is catastrophic - whereas mistakenly hitting Cmd-E (rather than Opt-E/Alt-E) simply opens up the search bar and has much less serious consequences. 2) E for Evaluate makes much more sense than W for Evaluate, Alt-E is easier to remember and is consistent with the Impromptu text editor shortcuts for all we long-time live coders.

**Comment by noahingham** [comment]
Good point. I haven't found a way to prevent the default Alt-e behaviour in Atom for Mac (inserting an accent "´") and Atom doesn't seem to like Ctrl-e either, so for now I've changed it to "Alt-s" to fix the closeness Cmd-w. This is a temporary solution, until I either figure out how to fix this or something better is suggested.

**Comment by toadhats** [comment]
I like alt-s, as it matches the muscle memory of saving a file after a change, and it's conceptually very much like saving a file, as it's updating what is 'saved' in the extempore instance - and accidentally pressing ctrl/cmd-s is harmless, if not especially useful in a live context. I'm not sure it needs to be changed again.
